tasks/shelter/README.md

```diff
@@ -296,334 +296,334 @@ Pay attention to additions:
+- if when resizing the browser window from the maximum to 320px a horizontal scrollbar appears, or the content goes off the screen, **-20** is removed.
+- if the horizontal scrollbar does not appear when the browser window is resized from the maximum to 320px, the content does not go off the screen, but some other layout problems appear (other than those in Google Chrome) - ** -5 is removed for each** with a description of the problem, but **no more than -20**.
 
-  Максимальное количество баллов, которое можно вычесть при наличии ошибок во время проверки в браузере **Mozilla Firefox** - **-20**.\*\*\*
+  The maximum number of points that can be deducted if there are errors during the check in the browser **Mozilla Firefox** - **-20**.\*\*\*
 
-  \* _на ширине экрана <768px за боковую границу экрана может заходить бургер-меню. Т.к. это нормальная практика, ошибкой это не считается, если при этом на странице не появляется горизонтальная прокрутка._  
-  \*\* _если во время общей проверки за страницу выставляется 0, дальнейшие критерии, относящиеся к этой странице, не проверяются._  
-  \*\*\* _дальнейшая проверка происходит только в браузере Google Chrome_
+  \* _when the screen width is <768px, the burger menu can go beyond the side border of the screen. Because this is normal practice, it is not considered an error if the horizontal scrolling does not appear on the page._
+  \*\* _if a page is set to 0 during the general check, no further criteria related to that page are checked._
+  \*\*\* _further check only happens in Google Chrome browser_
 
-### 1280px <= width (максимально -10 баллов за страницу)
+### 1280px <= width (maximum -10 points per page)
 
-При проверке на больших экранах допустимо:
-**1) фоны блоков растягиваются на всю ширину окна**, при этом сам контент находится в центре экрана и соответствует макету;
-**2) верстка занимает максимальную ширину 1280px** и соответствует макету, центрируется с равными отступами слева и справа, свободное пространство заполнено любым цветом;
+When checking on large screens, it is acceptable:
+**1) block backgrounds are stretched to the full width of the window**, while the content itself is in the center of the screen and corresponds to the layout;
+**2) the layout occupies a maximum width of 1280px** and matches the layout, is centered with equal indents on the left and right, the free space is filled with any color;
 
-Для любого из случаев выше **считается ошибкой**, если контент не центрируется, т.е. прижат к правой или левой стороне. В случае такой ошибки **снимается -10** баллов за страницу.
-Если контент центрирован, проверяется соответствие макету 1280px, для этого:
+For any of the cases above, it is **considered an error** if the content is not centered, i.e. aligned to the right or the left side. In case of such an error **removed -10** points per page.
```

Comment:
```suggestion
For any of the cases above, it is **considered an error** if the content is not centered, i.e. aligned to the right or the left side. In case of such an error **-10** points penalty applied per page.
```